generate/newlicense_test.go

```diff
@@ -148,56 +147,3 @@ func TestAppendAnonymizationKey(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, string(licAppended), string(licAppendedAgain))
 }
-
-func TestSplitPEM(t *testing.T) {
```

Comment:
`SplitPEM` was in `package license`, but its test was in `package generate`, which made no sense.

Since `SplitPEM` is not used externally, I unexported it and remove the test. It is an implementation detail that is covered by the existing `ParsePEM` tests.